source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCleanPathname, getCurrentBranch, getRepositoryInfo, upperCaseFirst} from '../github-helpers';
+
+function toggleHandler({delegateTarget: button}: delegate.Event): void {
+	const isHidden = select('.rgh-feature-screenshot')!.classList.toggle('d-none');
+	if (isHidden) {
+		button.textContent = 'View Screenshot';
+	} else {
+		button.textContent = 'Hide Screenshot';
+	}
+}
+
+async function init(): Promise<void | false> {
+	const currentFeature = location.pathname.split('/').pop()!.replace(/.tsx|.css/, '');
+	const featureInfo = features.featuresMeta.find(feature => feature.id === currentFeature)!;
```

Comment:
Just use the magic variable. This solution saves no bytes because they're in 2 different output files: options.js and refined-github.js

```suggestion
	const featureInfo = __featuresMeta__.find(feature => feature.id === currentFeature)!;
```